boudicca.base/publisher-event-html/src/main/resources/templates/events/event.hbs

```diff
@@ -1,48 +1,48 @@
-<div class="event">
-        <div class="event-image {{#if event.category}}event-image-{{event.category}}{{/if}}">
-            {{#if event.pictureUuid}}
-                <img src="/picture?uuid={{event.pictureUuid}}" style="height:100px;max-height:250px;max-width:250px;" alt="{{#if event.pictureAltText}}{{event.pictureAltText}}{{else}}Event Bild{{/if}}" aria-label="Event Bild" aria-labelledby="image" />
-            {{else}}
-                <svg title="Event Bild" viewBox="0 0 512 512" height="100px" >
-                    <use xlink:href="#{{#if event.category}}{{event.category}}{{else}}image{{/if}}"></use>
+<div class="event" role="button" aria-label="custom event element">
```

Comment:
i am not quite sure but i thought you should not make other elements like divs to buttons? and why did you add this anyway?

Reply:
yes, you are right, I was experimenting but it's a bad practice. anyway, I'll leave the event element like as it is.

Reply:
but the concern about the screen reader remains as long as the design (dom order) is as it is.

Reply:
reverted.